content/blog/2018-03-15-update-on-async-rendering.md

```diff
@@ -0,0 +1,168 @@
+---
+title: Update on Async Rendering
+author: [bvaughn]
+---
+
+For the past few months, the React team has been working on [asynchronous rendering](/blog/2018/03/01/sneak-peek-beyond-react-16.html), and we are very excited about the new features it enables.
```

Comment:
Do you think we could rewrite this paragraph to make the link to the talk more prominent? We could say that we've worked for more than a year to implement asynchronous rendering, and a month ago we unveiled the exciting features it unlocks in a talk with live demos. Not suggesting this particular wording, but I wonder if we should imply more strongly that `asynchronous rendering` links not just to some definition, but to demos of things that were never possible before, and that improve both UX and DX.

Reply:
Sounds like a good idea.